src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+                        String taskName = strArr[1];
+                        ToDo toDoTask = new ToDo(taskName);
+                        String addConfirmation = tl.addTask(toDoTask);
+                        System.out.println(addConfirmation);
+                        userInput = sc.nextLine();
+                    } else if (userInput.startsWith("deadline")) {
+                        String[] strArr = userInput.split(" ", 2);
+                        String[] strArr2 = strArr[1].split("/");
+                        if (strArr2.length != 2) {
+                            throw new IncorrectArgumentsException(userInput, 2, strArr2.length);
+                        }
+                        String taskName = strArr2[0];
+                        String dueTime = strArr2[1];
+                        Deadline deadlineTask = new Deadline(taskName, dueTime);
+                        String addConfirmation = tl.addTask(deadlineTask);
+                        System.out.println(addConfirmation);
+                        userInput = sc.nextLine();
+                    } else if (userInput.startsWith("event")) {
+                        String[] strArr = userInput.split(" ", 2);
+                        String[] strArr2 = strArr[1].split("/");
+                        if (strArr2.length != 3) {
+                            throw new IncorrectArgumentsException(userInput, 3, strArr2.length);
+                        }
+                        String taskName = strArr2[0];
+                        String startTime = strArr2[1];
+                        String endTime = strArr2[2];
+                        Event eventTask = new Event(taskName, startTime, endTime);
+                        String addConfirmation = tl.addTask(eventTask);
+                        System.out.println(addConfirmation);
+                        userInput = sc.nextLine();
+                    } else if (userInput.startsWith("delete")) {
+                        String[] strArr = userInput.split(" ", 2);
+                        int taskNumber = Integer.parseInt(strArr[1]);
+                        if (taskNumber < 1 || taskNumber > tl.items) {
+                            throw new InvalidTaskException(taskNumber);
+                        }
+                        String deleteConfirmation = tl.deleteTask(taskNumber - 1);
+                        System.out.println(deleteConfirmation);
+                        userInput = sc.nextLine();
+
+                    } else {
+                        throw new InvalidInputException(userInput);
+                    }
+                } else {
+                    String output = tl.printTaskList();
+                    System.out.println(output);
+                    userInput = sc.nextLine();
+                }
+            } catch (DukeException exception) {
+                System.out.println(exception);
+                userInput = sc.nextLine();
+            }
+        }
+
+        String farewell = duke.farewellUser();
```

Comment:
I think it is also possible to directly print the message within the method itself, instead of retrieving the string and printing it.

Also, I think if you want to get the farewell message for the user, I believe the naming convention should be "getFarewellUser()" which will be more comprehensible.